dotnet/TimePlugin/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Do we need separate project for this example? Can we add it as integration test under `Concepts` folder? If the point is to be able to accept user input (like from Console in this case), maybe it's worth to add it to .NET notebooks?

Reply:
We already have, I'm adding this Demo as an ask from @sophialagerkranspandey, to support a blog post on how simple is to use a Plugin with Auto Function Calling with minimum setup effort.